elementary/utils/models.py

```diff
@@ -0,0 +1,26 @@
+from elementary.monitor.alerts.model import ModelAlert
+from elementary.monitor.alerts.source_freshness import SourceFreshnessAlert
+from elementary.monitor.alerts.test import ElementaryTestAlert, TestAlert
+
+
+def get_shortened_model_name(model):
+    if model is None:
+        # this can happen for example when a Singular test is failing for having no refs.
+        return None
+    return model.split(".")[-1]
+
+
+def alert_to_concise_name(alert):
```

Comment:
When I am thinking of it, this is not a model util but a property of Alert.
So I think that each Alert should have its own concise_name property.
Lets move each instance to its right class

Reply:
Accept

Reply:
done